trello_api.py

```diff
@@ -0,0 +1,92 @@
+from flask import session
+from classes import todoStatus, todoItem
+import os, requests, json, logging
+
+API_PREFIX = 'https://api.trello.com/1/'
+
+API_KEY = os.getenv('API_KEY')
+API_TOKEN = os.getenv('API_TOKEN')
+# API_PARAMS was getting appended with additional parameters, so had to set variable in each API function
+API_PARAMS = {'key': API_KEY, 'token': API_TOKEN}
+board = os.getenv('BOARD_ID')
+headers = {"Accept": "application/json"}
+
+
+def trelloGet(trelloPath):
+    API_PARAMS = {'key': API_KEY, 'token': API_TOKEN}
+    return requests.get(API_PREFIX + trelloPath, params=API_PARAMS).json()
+
+
+def get_trello_lists():
+    trelloLists = []
+    for item in trelloGet(f'boards/{board}/lists'):
+        list_data = todoStatus(
+            item['id'],
+            item['name']
+        )
+        trelloLists.append(list_data)
+        if list_data.status == "Not Started":
+            session['newItemId'] = list_data.trelloId
+        if list_data.status == "In Progress":
+            session['progressId'] = list_data.trelloId
+        if list_data.status == "Completed":
+            session['completedId'] = list_data.trelloId
+    return trelloLists
+    
+
+def get_trello_cards():
+    trelloCards = []
+    trelloLists = get_trello_lists()
+    for todoList in trelloLists:
+        for item in trelloGet(f'lists/{todoList.trelloId}/cards'):
+            todo = todoItem(
+                item['id'],
+                item['name'],
+                item['desc'],
+                todoList.status
+            )
+            trelloCards.append(todo)
+    return trelloCards
+
+
+def trelloPost(title, description):
```

Comment:
Try to stick to the snake_case naming convention rather than camelCase. It's just python thing to use snake_case, but you should always be consistent and currently you're mixing cases.

Also, this function name isn't the clearest. It's not a generic function for making a POST request to Trello, it's specifically creating a new card.